### ISSUE TITLE
Create day/twilight/night modes

### ISSUE THREAD

**Issue by morgane** [nickbytes/nabu#8]
Day = white background, black text
Twilight = yellow background, black text
Night = black background, white text

(Similar to reading apps, etc)

Toggle somehow?

**Comment by nickbytes** [comment]
Settings panel @morgane ?

**Comment by morgane** [comment]
Yeah I think that'd be good. Could have settings for font size too

**Comment by morgane** [comment]
TODO: create designs for what each mode could look like
